recipes/argtable3/all/conanfile.py

```diff
@@ -88,22 +89,28 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "cmake"))
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
 
+        # These targets were for versions <= 3.2.0 (newer create argtable3::argtable3)
         target_name = "argtable3" if self.options.shared else "argtable3_static"
         self._create_cmake_module_alias_targets(
             os.path.join(self.package_folder, self._module_file_rel_path),
             {target_name: "argtable3::argtable3"}
         )
 
     def package_info(self):
-        self.cpp_info.libs = ["argtable3" if self.options.shared else "argtable3_static"]
+        suffix = ""
+        if tools.Version(self.version) >= "3.2.1":
+            if self.settings.build_type == "Debug":
+                suffix = "d"
+        self.cpp_info.libs = [("argtable3" if self.options.shared else "argtable3_static") + suffix]
```

Comment:
```suggestion
        if not self.options.shared:
            suffix += "_static"
        if tools.Version(self.version) >= "3.2.1":
            if self.settings.build_type == "Debug":
                suffix += "d"
        self.cpp_info.libs = ["argtable3{}".format(suffix)]
```

Maybe simplier like this?